coffeecard/CoffeeCard.Tests.Unit/Models/SemesterStartTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using CoffeeCard.WebApi.Models;
+using System;
+using Xunit;
+
+namespace CoffeeCard.Tests.Unit.Models
+{
+    public class SemesterStartTests
+    {
+        [Theory(DisplayName = "When called in months Jan-Jun (inclusive), GetSemesterStart returns the last Monday of January")]
+        [InlineData("2022/01/31", "2022/01/31")]
+        [InlineData("2023/01/31", "2023/01/30")]
+        [InlineData("2024/05/12", "2024/01/29")]
+        [InlineData("2025/06/24", "2025/01/27")]
+        [InlineData("2026/01/01", "2026/01/26")]
+        [InlineData("2027/01/01", "2027/01/25")]
+        [InlineData("2030/06/30", "2030/01/28")]
```

Comment:
Since we are testing the method, it would be great to create a separate test for the case of the autumn semester.

Reply:
Added and changed the display name of the theory.